docs/es/advanced/navigation-guards.md

```diff
@@ -32,8 +32,18 @@ Cada función guardia recibe tres argumentos:
 
   - **`next('/')` o `next({ path: '/' })`**: redirecciona a una ruta diferente. La navegación actual será abortada y una nueva será iniciada.
 
+  - **`next(error)`**: (2.4.0+) si el argumento pasado a `next` es una instancia de `Error`, la navegación se abortará y el error será pasado a las _funciones callback_ registradas a través de `router.onError()`.
+
 **Asegúrese de llamar siempre a la función `next`, sino el _hook_ nunca será resuelto.**
 
+### Guardias de resolución globales
+
+> Nuevo en 2.5.0
+
+A oartir de la versión 2.5.0 puedes registrar un guardia global con `router.beforeResolve`. Esto es similar a `router.beforeEach`, con la diferencia que los guardias de resolución serán lamados justo antes de que la navegación sea confirmada, **después que todos los guardias en el componente y los componentes de rutas asíncronos sean resueltos**.
+
+### 'Hooks after' globales
```

Comment:
no se si algo como Post _hooks_ globales suena mejor.
Es mejor ponerlo in cursiva como en las otras partes